.prettierrc.json

```
@@ -0,0 +1,3 @@
+{
+    "echo": {"singleQutore": true}
```

The `echo` command is a zsh/bash command and was meant to be run from the command line. The result would be that this `JSON` would be added to the `.prettierrc.json` file"

```suggestion
   {
     "singleQuote": true
   }
```